nilearn/datasets/tests/data/list_archive_contents.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
```

Comment:
It should be.

All the workshops I  have been a part of in France, including at NS & Marseille, I was the one primarily doing the troubleshooting on audience systems while other team members were presenting the demo material. I got a close look at all the systems used by the audience. At least a third of them used Windows.

This is not a super difficult problem though. There are automated tools which accept a few questions and generate a Powershell script, which we can include.

The default installation of Git for Windows includes GitBash, which will run these scripts. At the very least, we should mention this route.

Reply:
Thanks. if we want it to work on Windows it is easy to rewrite it in Python.

However I think we can just remove it; I used a script because I had to do it
for many archives but it is not hard to do manually: we just need to download
the archive, list its contents and write them to a file (one item per line), and
add the url (and optionally archive format) at the top of the file. I had added
the script to show how the existing files were generated and because it
describes what needs to be done better than plain english but it is not really
necessary. same for `nyu_file_contents.sh`

Reply:
If the helper script is removed, I'd also make sure references are removed in the (very helpful !) dosctring for the Sender class, like [this one](https://github.com/jeromedockes/nilearn/blob/4f37b4162c6e6b3e48a6c96b5253ebc0aa85a7ca/nilearn/datasets/_testing.py#L176).

Reply:
for now, as a compromise between what Gael and you suggested, I removed the NYU one and kept the other. we can revisit once the "contributing" docs on adding datasets are written